getting_started/step_by_step/your_first_game.rst

```diff
@@ -516,15 +499,17 @@ starting a new game.
     func start(pos):
         position = pos
         show()
-        $CollisionShape2D.disabled = false
+        # Must be deferred as we can't change physics properties on a physics callback.
+        $CollisionShape2D.set_deferred("disabled", false)
```

Comment:
This one doesn't need to be deferred, because it's not called during the physics callback.

Reply:
Yes, you're right. This slipped in with a change to the demos and I missed it.